src/zrc/conf/docker.py

```diff
@@ -41,6 +41,11 @@ def getenv(key, default=None, required=False, split=False):
     # https://github.com/jazzband/django-axes/blob/master/docs/configuration.rst#cache-problems
     'axes_cache': {
         'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+    },
+    # Cache for BIO removal sync with DRC
+    'drc_sync': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
```

Comment:
These files are not shared between different pods in kubernetes and we don't have a storage class that provides this currently. Can you use Redis instead?